### DISCUSSION TITLE
Scanning packages for Tailwind classes

### DISCUSSION THREAD

**Discussion by SimeonGriggs** [windicss/windicss#74] [Q&A] [answered]
I have a grid overlay I use between projects and publish to `npm`, but the class names it uses aren't being picked up by WindiCSS. Is there a way to point WindiCSS at a component or package?

**Comment by voorjaar** [comment, marked as answer]
https://github.com/windicss/windicss/issues/100

**Reply by dduupp** [reply]
@voorjaar How does #100 relate to this question exactly?

I'm running into the same problem, my UI components have TailwindCSS classes, but come from a different package, something like this:

```html
<template>
  <r-section>
    Hello there
    <r-button>Click me!</r-button>
  </r-section>
</template>

<script>
import { RButton, RSection } from 'my-components';

import 'virtual:windi.css';

export default {
  components: {
    RButton,
    RSection,
  },
};
</script>
```

The TailwindCSS classes from `<r-button>` and `<r-section>` get purged and these components don't get styled.

**Reply by antfu** [reply]
Maybe you are meaning this https://github.com/windicss/vite-plugin-windicss/issues/86

**Comment by hannoeru** [comment]
Which plugin are you using? And can you provide more context?
